Ts search embedd factory2

lizozom opened the pull request:
## Summary

Search Embeddable to TS

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

- [ ] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)
- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios
- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)

Commit:
merge

Commit:
Promise -> async await

Commit:
object desctructure

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/11568/)

epixa committed:
Move src/deprecation to src/legacy/deprecation

epixa committed:
Update paths to ui, server, deprecation, plugin_discovery in src/legacy

This commit accompanies the four that precede it. Rather than squash
them altogether, the four previous commits all do nothing except move
files to help avoid conflicts.

epixa committed:
Move src/utils to src/legacy/utils

epixa committed:
Move src/plugin_discovery to src/legacy/plugin_discovery

epixa committed:
Move src/server to src/legacy/server

epixa committed:
Move src/ui to src/legacy/ui

Commit:
Merge branch 'master' into ts-search-embedd-factory2

elasticmachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pull-request/12178/)

sorenlouv committed:
[APM] Cleanup ES queries (#30363)

cchaos committed:
Homepage style updates (#30341)

* Better responsive handling of home grids

* Update sample data images to match UI updates

* Hide non-existent MSSQL logo so it doesn’t throw errors

* Dark mode images for sample data

* Som layout fixes to tutorials

* remove unused file

* convert synopsis to use EuiCard

* Snaps

* IE fix

* Remove unnecessary inline margin for flex group spacing

* More snaps cuz of the IE fix

formgeist committed:
[APM] Fix Stackframe fontsize (#30489)

Decreased the font size in the code box to improve legibility.

loixlab committed:
[CCR] Put back API integration test for follower indices (#30260)

flash1293 committed:
Add typings for saved_object_finder (#30067)

loixlab committed:
[Watcher] Add Index, HipChat, PagerDuty, and Jira Action types on the client (#30043)

formgeist committed:
[APM] Replace colour variables (#30491)

Replaced APM UI custom colour variables with EUI theme variables.

peteharverson committed:
[ML] Removes non ECS apache, nginx and auditbeat data recognizer modules (#30537)

azasypkin committed:
Add `zh-CN.json` translations and respective compatibility checks via i18n tools (#30378)

lukasolson committed:
Fix buildEsQuery to ignore filters if not in index (#29880)

* Fix buildEsQuery to ignore filters if not in index

* Fix tests and move getEsQueryConfig to package

* Revert changes to TSVB

* Remove console log

* Review feedback

Commit:
Track all test times (#30285)

* track all test times

* Add full title for easy ingestion to track performance times

epixa committed:
Move more directories into src/legacy (#30435)

Moves the following directories into src/legacy:

src/deprecation         -> src/legacy/deprecation
src/ui                         -> src/legacy/ui
src/server                  -> src/legacy/server
src/plugin_discovery -> src/legacy/plugin_discovery
src/utils                     -> src/legacy/utils

tylersmalley committed:
[docs] Improve contributing docs on runing Elasticsearch (#30286)

Closes #28657 #23692

Signed-off-by: Tyler Smalley <tyler.smalley@elastic.co>

joshdover committed:
Bump max_old_space_size in CI (#30653)

lcawl committed:
[DOCS] Removes X-Pack settings section (#30554)

cjcenizal committed:
Fix ILM dependency upon xpack.index_management.enabled setting. (#30592)

jen-huang committed:
Fix ordering of CCR plugin (#30596)

Commit:
[ftr/uiSettingDefaults] disable telemetry by default (#30473)

Bargs committed:
skip flaky dashboard filtering test (#30682)

lcawl committed:
[DOCS] Sorts the list of Kibana settings (#30551)

sorenlouv committed:
[APM] Fix progress indicator (#30649)

justinkambic committed:
[Uptime] Fix snapshot query to handle large numbers of monitors (#30441)

* Refactor snapshot histogram ES query and subsequent processing.

* Fix issues on client, remove obsolete code, update tests.

* Get type check passing and update Snapshot component's unit tests to work with new schema data.

mattapperson committed:
[BeatsCM] remove type from ES template and requests (#30549)

* remove dype from ES template and requests

* remove doc type?

jgowdyelastic committed:
[ML] Fixing detector help link in advanced wizard (#30641)

* [ML] Fixing detector help link in advanced wizard

* tiny refactor

thomasneirynck committed:
[Maps] Fix and improve error messaging when adding TMS from kibana.yml (#30529)

cqliu1 committed:
Added isFirstLoad to transient state. (#30150)

Commit:
[esArchiver] stable archives (#30477)

* [esArchiver/save] serialize with stable json stringify

* [esArchives] rebuild all archives

Commit:
[esArchiver] only load uiExports for xpack when necessary (#30474)

* [esArchiver] only load uiExports for xpack when necessary

* [esArchiver/kibanaIndex] remove caches, node caches modules so uiExports shouldn't be too bad

* [esArchiver/cli] include command name in error

* [esArchiver/kibana] include error in new message

* remove comment about caching

mattapperson committed:
Revert "[BeatsCM] remove type from ES template and requests (#30549)" (#30728)

This reverts commit 93a10a1c395d87a3853f5484de0cd80aee5692b3.

mikecote committed:
Clone saved object document before migrating to prevent accidental mutations (#30475)

* Clone saved object document before migrating to prevent accidental mutations

* Add tests

* Add example comment

* Use latest version of lodash cloneDeep

* Revise spaces functional tests to use same sample data.

* Fix broken test

justinkambic committed:
Remove double-format for duration value. (#30664)

Commit:
[ftr/services/es] use apiVersion from es plugin (#30733)

Commit:
[kuery] support matching field names with newlines (#29539)

* [kuery] support matching field names with newlines

* Add support for newlines in KQL grammar

* Add test for newlines in wildcards

Commit:
Revert "[kuery] support matching field names with newlines (#29539)"

This reverts commit 66362856f625d0679552799cbd83e528130082b1.

nreese committed:
[Maps] remove toasts for layer add and layer load error (#30584)

* [Maps] remove toasts for layer add and layer load error

* remove side-effects of LAYER_DATA_LOAD_ERROR action

* more cleanup

nreese committed:
[Maps] properly wait for layers to load in functional tests to avoid flakiness (#30580)

* [Maps] wait for maps to load in functional tests to avoid flakyness

* add wait when manually accessing URL from browser service

* handle join source with missing index pattern id

* update triggerSingleRefresh to waitForLayersToLoad

* fix jest test

* add layer error test for KibanaTilemapSource with out map.tilemap.url configuration

* remove isInErrorState and errorMessage from layerslayerListJSON since it should not have been put there in first place

flash1293 committed:
Add request timeout for vega requests (#30482)

jgowdyelastic committed:
[ML] General ML style tweaks for K7 compatibility (#30536)

* [ML] [WIP] General ML style tweaks for K7 compatibility

* file data viz clean up

ppisljar committed:
fixing vis filters to ignore invalid entries (#30633)

alexwizp committed:
Fix Std. Deviation aggregation crashes TSVB

flash1293 committed:
Expand add panels flyout and increase default rows per page (#30508)

simitt committed:
[APM] Set Index Pattern version as string. (#30796)

fixes #30651

timroes committed:
Fix i18n documentation (#30818)

bmcconaghy committed:
refactoring server routes in index management to use common code (#30299)

* refactoring server routes in index management to use common code

* streamlining per PR comments

Commit:
skip flaky filtering test (#30840)

simianhacker committed:
Fixing font size for icon div (#30744)

Commit:
Merge branch 'ts-search-embedd-factory2' of github.com:lizozom/kibana into ts-search-embedd-factory2
